internal_setup.bzl

```diff
@@ -35,9 +34,6 @@ def rules_python_internal_setup():
 
     bazel_skylib_workspace()
 
-    rules_proto_dependencies()
```

Comment:
For workspace builds, shouldn't there be some sort of proto-setup call still? Did I miss where it is?

Reply:
There's a `protobuf_deps` call just after. We only need to setup protobuf, now that the rules are moving there.